openedx/core/djangoapps/contentserver/middleware.py

```diff
@@ -156,7 +156,8 @@ def process_request(self, request):
                         if 0 <= first <= last < content.length:
                             # If the byte range is satisfiable
                             response = HttpResponse(content.stream_data_in_range(first, last))
-                            response['Content-Range'] = b'bytes {first}-{last}/{length}'.format(
+                            # pylint: disable=unicode-format-string
+                            response['Content-Range'] = six.u('bytes {first}-{last}/{length}').format(
```

Comment:
You can just prefix with a `u'bytes {first}-{last}/{length}'` instead which is a lot simpler and cleaner.